lang/cs/Org.Apache.REEF.Network/Group/Pipelining/Impl/DefaultPipelineDataConverter.cs

```diff
@@ -0,0 +1,74 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using System.Collections.Generic;
+using Org.Apache.REEF.Tang.Implementations.Tang;
+using Org.Apache.REEF.Tang.Interface;
+using Org.Apache.REEF.Tang.Annotations;
+
+
+namespace Org.Apache.REEF.Network.Group.Pipelining.Impl
+{
+
+    /// <summary>
+    /// Default IPipelineDataConverter implementation
+    /// This basically is a non-pipelined implementation that just packs the whole message in one single PipelineMessage
+    /// </summary>
+    /// <typeparam name="T">The message type</typeparam>
+    public class DefaultPipelineDataConverter<T> : IPipelineDataConverter<T>
+    {
+        [Inject]
+        public DefaultPipelineDataConverter()
```

Comment:
Add the following for IPipelineDataConverter:
[DefaultImplementation(typeof(DefaultPipelineDataConverter<>))]
public interface IPipelineDataConverter<T>